openmetadata-service/src/main/java/org/openmetadata/service/util/JsonUtils.java

```diff
@@ -193,40 +192,15 @@ public static <T> T convertValue(Object object, TypeReference<T> toValueTypeRef)
   public static JsonValue applyPatch(Object original, JsonPatch patch) {
     JsonStructure targetJson = JsonUtils.getJsonStructure(original);
 
-    //
-    // -----------------------------------------------------------
-    // JSON patch modification 1 - Reorder the operations
-    // -----------------------------------------------------------
-    // JsonPatch array operations are not handled correctly by johnzon libraries. Example, the
-    // following operation:
-    // {"op":"replace","path":"/tags/0/tagFQN","value":"User.BankAccount"}
-    // {"op":"replace","path":"/tags/0/labelType","value":"MANUAL"}
-    // {"op":"remove","path":"/tags/1"}
-    // {"op":"remove","path":"/tags/2"}
-    // Removes second array element in a 3 array field /tags/1
```

Comment:
I suggest adding a test case for this scenario. This seems like a contentious issue that we should track for regressions if we decide to handle it differently in the future.

Reply:
Hey, do you mean testing that we are raising an error fi the operation comes on the wrong order? Didn't get it.

Thanks!